### DISCUSSION TITLE
Json from collection data file returns [object Object]

### DISCUSSION THREAD

**Discussion by psvaibhav** [thunderclient/thunder-client-support#1106] [Q&A] [answered]
I need to supply a series of json payloads to run a collection. I use a variable in the request body, say {{location}}, and the value for this variable is supplied by json arrray of 'location' jsons in the data file. 

The data file contents - [{"location":{"city":"Chicago"}},{"location":{"city":"Los Angeles"}},{"location":{"city":"New York"}}]

When I run the collection, I get an [object Object] in the body. How can I get the json string {"city":"Los Angeles"} in the body? {{location | stringify}} did not help.
![Screen Shot 2023-04-07 at 12 25 57 PM](https://user-images.githubusercontent.com/12549340/230652613-b9d44c21-efa7-4ef2-aef3-6a7a18d91503.png)

**Comment by rangav** [comment, marked as answer]
Hi @psvaibhav Currently the values of variables is expected not to be objects.

Example file data
```json
[
  {
    "location": "Chicago",
     "property2": "value1"
  },
  {
    "location": "Los Angeles",
     "property2": "value2"
  },
  {
    "location": "New York",
     "property2": "value2"
  }
]
```
